openlcs/libs/sc_handler.py

```diff
@@ -0,0 +1,154 @@
+import koji
+import glob
+import os
+import uuid
+import sys
+
+# Fix absolute import issue in openlcs.
+openlcs_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+if openlcs_dir not in sys.path:
+    sys.path.append(openlcs_dir)
+from libs.common import compress_source_to_tarball  # noqa: E402
+
+
+class SourceContainerHandler(object):
+    """
+    Object used for handle source in source containers.
+    """
+    def __init__(self, config=None):
+        self.config = config
+
+    def get_container_component(self, sc_nvr, release=None):
+        """
+        Get container component.
+        """
+        nvr = koji.parse_NVR(sc_nvr)
+        if not release:
+            name, version, release = (nvr.get('name'),
+                                      nvr.get('version'),
+                                      nvr.get('release'))
+        container_component = {
+                'uuid': str(uuid.uuid4()),
+                'name': name,
+                'version': version,
+                'release': release,
+                'type': 'CONTAINER_IMAGE',
+                'nvr': sc_nvr,
+                'arch': '',
+                'license': ''
+        }
+        return container_component
+
+    def get_rpm_components(self, srpm_dir, sc_nvr):
```

Comment:
The component in  `get_container_component `, `get_rpm_components ` and  `get_misc_component ` have same parameters, so it is better to define them in one function, such as https://github.com/RedHatProductSecurity/openlcs/blob/feecf6c95c96c8468d5b072a40ab7843a7674208/openlcs/libs/corgi_handler.py#L23

Reply:
Updated.